packages/chrome/src/elements/nav/NavItem.tsx

```diff
@@ -44,6 +46,7 @@ export const NavItem = React.forwardRef<HTMLButtonElement, INavItemProps>(
         hue={hue}
         isDark={isDark}
         isLight={isLight}
+        aria-current={ariaCurrent}
```

Comment:
```suggestion
        aria-current={other.isCurrent}
```

We can't guarantee that these nav items are always used for page level navigation.

Reply:
Gotcha. How do consumers add destinations (`href`s) to these elements? Is there an opportunity to conditionally set the value to `page` if we know the components are intended to behave like links?

Reply:
Fair enough. Is there someplace we can document this, so that consumers can see that this is the preferred approach?